README.md

```diff
@@ -72,7 +70,7 @@ $ bin/setup
 $ bin/rake
 
 # Fire up the app and open it in a browser
-$ heroku local
+$ yarn start
```

Comment:
The app template now recommends `yarn start`. See: https://github.com/carbonfive/raygun-rails/blob/main/README.md#running-the-application-locally